iam-persistence/src/main/resources/db/migration/mysql/V103__add_active_and_status_changed_on_to_client_details.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE client_details ADD COLUMN (active BOOLEAN, 
+                                        status_changed_on TIMESTAMP DEFAULT '1970-01-01 00:00:01',
```

Comment:
Why not a CURRENT_TIMESTAMP as default? First time column active is added is officially the first "status change". This value is relevant if used together with status_changed_by and in case active is false. What do you think?

Reply:
Make sense. Updated.